### ISSUE TITLE
Streamline setup+use

### ISSUE THREAD

**Issue by DTrejo** [socketio/socket.io#65]
I've been trying use socket.io for a project of mine, and I've noticed a few things. Some of these are probably just me being silly, but here goes:
- version of socket.io-node in npm is not .6
- this git repo has tons of sub-repos and lib folders — trying to clone the .6 version and figure out how things work can be confusing
- It is unclear to me how this combines with the connect framework [update: just use it like normal]
- When I fire up my server, and bind socket.io-node to it, my client does not connect, but I don't receive any error messages anywhere so I don't know what's going on. [update: make sure port numbers are same on client and server]

So my thoughts are that #1, it would be nice to have the latest version on npm, so that no mistakes are caused by downloading the .6 client script and using the .5.3 server version.

tldr; It'd be nice to have the latest version in npm.

Have a good day,
David
